lib/commands/exam.js

```diff
@@ -124,35 +129,95 @@ module.exports = TestCommand.extend({
    * @override
    */
   _generateCustomConfigs: function(commandOptions) {
-    var config = this._super._generateCustomConfigs.apply(this, arguments);
-
-    if (this.validator.shouldParallelize) {
-      // Get the testPage from the generated config or the Testem config and
-      // use it as the baseUrl for the parallelized test pages
-      var baseUrl = config.testPage || this._getTestPage(commandOptions.configFile);
-      var count = commandOptions.split;
-      var partitions = commandOptions.partition;
-
-      if (!partitions) {
-        partitions = [];
-        for (var i = 0; i < commandOptions.split; i++) {
-          partitions.push(i + 1);
-        }
-      }
+    const config = this._super._generateCustomConfigs.apply(this, arguments);
+    const customBaseUrl = this._getCustomBaseUrl(config, commandOptions);
+    const shouldParallelize = this.validator.shouldParallelize;
+    const shouldLoadBalance = this.validator.shouldLoadBalance;
 
-      if (Array.isArray(baseUrl)) {
-        var command = this;
-        config.testPage = baseUrl.reduce(function(testPages, baseUrl) {
-          return testPages.concat(command._generateTestPages(baseUrl, partitions, count));
-        }, []);
-      } else {
-        config.testPage = this._generateTestPages(baseUrl, partitions, count);
+    if (!shouldParallelize && !shouldLoadBalance) {
+      return config;
+    }
+
+    let ids = commandOptions.partition;
+    let appendingParam = '_partition';
+
+    if (shouldParallelize && typeof ids == 'undefined') {
+      ids = [];
+      for (var i = 0; i < commandOptions.split; i++) {
+        ids.push(i + 1);
       }
+      ids = partitions;
+    } else if (shouldLoadBalance) {
+      let browserCount = commandOptions.loadBalance;
+      appendingParam = '_browser';
+      if (!browserCount) {
+        browserCount = 1;
```

Comment:
If we default to 1 when no browser count is defined, it might not register to the user that they can set a browser count value.
I feel we should be explicit to notify the user that this option requires a value to be passed.